### DISCUSSION TITLE
increase "maxRoutes" default settings

### DISCUSSION THREAD

**Discussion by Sriinspire** [harlan-zw/unlighthouse#162] [Q&A]
How to  increase "maxRoutes" default settings of 200 to more, i need 2800 minimum routes to be scanned
i am pretty new to unlighthouse..can any one help me out

**Comment by hacknug** [comment]
You should be able to do so using [`scanner.maxRoutes`](https://unlighthouse.dev/api/config#scannermaxroutes) as explained in the docs 👍
